modules/core/src/frame.cpp

```diff
@@ -29,7 +29,7 @@
     SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
-#include "frame.h"
+#include "../include/frame.h"
```

Comment:
I'm assuming this is just something you missed when you removed fixed the other files, but this isn't a necessary change.